app/access/models.py

```diff
@@ -2,9 +2,18 @@
 
 from django.db import models
 from django.db.models.base import ModelBase
+from django.utils import timezone
 from django.utils.translation import pgettext_lazy as _
 
 
+class ActiveUserManager(models.Manager["User"]):
+    """ActiveUserManager filters out any users that have deleted_at set.
```

Comment:
I would avoid using variable names in comment when possible. Variable names change and then often the comments are out-of-sync. How about "ActiveUserManager filters out any users that marked as disabled." instead?